docs/hooks/useTransactor.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# useTransactor
+
+Use this hook to interact with the chain and get UI feedback on the transaction status. You can pass in anything that is a valid parameter to [Viem's `sendTransaction` function](https://viem.sh/docs/actions/wallet/sendTransaction#parameters).
+
+```ts
+const transactor = useTransactor();
+const writeTx = transactor({
+  to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  value: 1000000000000000000n,
+});
+await writeTx();
+```
+
+This example tries to send 1 ETH to the address `0x70997970c51812dc3a010c7d01b50e0d17dc79c8`, prompting the connected `WalletClient` for a signature. And in the case of a successful transaction, it will show a popup in the UI with the message: "🎉 Transaction completed successfully!".
```

Comment:
Regarding : "🎉 Transaction completed successfully!"

I think it would be nice we add gif showing transaction loading and success, similar to what we have at [`AddressInput`](https://docs.scaffoldeth.io/components/AddressInput) or Balance component.

Reply:
https://github.com/scaffold-eth/se-2-docs/commit/311d891c399f3963813d78c4efa89560e663c366